src/Components/DataEntry/TextArea.vue

```diff
@@ -1,70 +1,61 @@
 <template>
-	<div class="c-application c-textarea" :class="[computedType]">
-		<textarea
-			ref="input"
-			v-model="sync_value"
-			:placeholder="placeholder"
-			:style="inputStyle"
-			@input="$emit('input', $event.target.value)"
-		/>
+	<div class="c-application c-textarea" :class="[computedType]" :style="[computedHeight]">
+		<textarea ref="textarea" :value="value" :placeholder="placeholder" :readonly="readOnly" @input="resize" />
 	</div>
 </template>
 
 <script>
+export const textAreaTypes = ['basic', 'outline', 'reply'];
 export default {
 	name: 'TextArea',
 	props: {
-		// textarea placeholder 문구
 		placeholder: {
 			type: String,
-			default: null,
+			default: 'placeholder',
```

Comment:
```suggestion
			default: '내용을 입력해주세요.',
```
디자이너분들이 **placeholder**로 만들어 놓았어도, 우리는 실제로 사용할 때를 상상하면서 만들면 좋을 것 같아요